src/common/addlayers/RegistryLayersDirective.js

```diff
@@ -183,9 +183,14 @@
               }
             }
 
+            scope.toggleCart = function(layerConfig) {
```

Comment:
Is `toggleCart` a good name for this method?
It does more than just to "toggle" the layer, right?
Why does it reset the previewLayers to just the one that is toggled? Shouldn't all layers be visible that are in the cart?